patches/tModLoader/Terraria/GameContent/UI/BigProgressBar/BigProgressBarSystem.TML.cs

```diff
@@ -0,0 +1,9 @@
+namespace Terraria.GameContent.UI.BigProgressBar
+{
+	public partial class BigProgressBarSystem
+	{
+		//If any mods need to fetch special vanilla bars
+		public bool TryGetSpecialVanillaBossBar(int netID, out IBigProgressBar bossBar) =>
+			_bossBarsByNpcNetId.TryGetValue(netID, out bossBar);
```

Comment:
This is much more preferable
```suggestion
		//If any mods need to fetch special vanilla bars
		public bool TryGetSpecialVanillaBossBar(int netID, out IBigProgressBar bossBar)
			=> _bossBarsByNpcNetId.TryGetValue(netID, out bossBar);
```

Reply:
adjusted and added docs aswell